QuestionBot/QuestionBot.UnitTests/Model/InMemoryStoreTests.cs

```diff
@@ -0,0 +1,114 @@
+        }
+
+        [Test]
+        public void New_question_creates_record(){
+            string question = "What is 1+2?";
+
+            IRecord questionRecord = _storeTest.CreateRecord(question);
+
+            Assert.AreEqual(questionRecord.Question, question);
+            Assert.AreNotEqual(questionRecord.TimeAsked, default(DateTime));
+            Assert.AreNotEqual(questionRecord.ID,default(int));
+        }
+
+        [Test]
+        public void Updating_question_with_answer_stores_my_answer(){
+            string question = "What is 1+2?";
+            string answer = "3";
+            IRecord answerRecord;
+
+            IRecord questionRecord = _storeTest.CreateRecord(question);
+            bool updatedRecord = _storeTest.TryUpdateRecord(questionRecord.ID, answer,out answerRecord);
+            _retrievedRecords = _storeTest.GetRecords();
+            IRecord myRecord = _retrievedRecords.ElementAt(0);
+
+            Assert.IsTrue(updatedRecord);
+            Assert.AreEqual(myRecord.Question, question);
+            Assert.AreEqual(myRecord.Answer, answer);
+            Assert.AreEqual(myRecord.TimeAsked, questionRecord.TimeAsked);
+            Assert.AreEqual(myRecord.TimeAnswered, questionRecord.TimeAnswered);
+            Assert.AreEqual(myRecord.ID, questionRecord.ID);
+
+            Assert.AreEqual(answerRecord.Question, question);
+            Assert.AreEqual(answerRecord.Answer, answer);
+            Assert.AreEqual(answerRecord.TimeAsked, questionRecord.TimeAsked);
+            Assert.AreEqual(answerRecord.TimeAnswered, questionRecord.TimeAnswered);
+            Assert.AreEqual(answerRecord.ID, questionRecord.ID);
+        }
+
+        [Test]
+        public void GetRecords_returns_all_question_records(){
+            string question1 = "What is 1+2";
+            string question2 = "What is 2+3?";
+            string answer1 = "3";
+            string answer2 = "five";
+            IRecord answerRecord;
+
+            IRecord record1 = _storeTest.CreateRecord(question1);
+            IRecord record2 = _storeTest.CreateRecord(question2);
+            IEnumerable<IRecord> questionRecords = new List<IRecord>{record1,record2};
+
+            _storeTest.TryUpdateRecord(record1.ID, answer1, out answerRecord);
+            _storeTest.TryUpdateRecord(record2.ID, answer2, out answerRecord);
+
+            _retrievedRecords = _storeTest.GetRecords();
+
+            CollectionAssert.AreEqual(questionRecords, _retrievedRecords);
+        }
+
+        [Test]
+        public void GetRecords_returns_empty_list_if_there_are_no_records(){
+            _retrievedRecords = _storeTest.GetRecords();
+
+            Assert.IsEmpty(_retrievedRecords);
+        }
+
+        [Test]
+        public void Updating_record_with_blank_answer_returns_false(){
+            string question1 = "What is 1+2";
+            string question2 = "What is 2+3?";
+            string answer1 = "";
+            string answer2 = null;
+            IRecord x;
```

Comment:
super minor: it might be more clear if you named this variable (and all those like it) `unused` or something